### ISSUE TITLE
Edit the Firebase Database

### ISSUE THREAD

**Issue by colbySullivan** [MuellMark/Course-Scheduler#159]
* Fix the issue with limited read and writes
* Look into limiting who can overwrite new classes
* Limit the amount each person can write individually
* Rename the folders

**Comment by colbySullivan** [comment]
I spent the last 30 minutes reading documentation on the rule page of the Database. I attached the current rules in the picture below and the documentation we can use for user authentication is we decide to do so.
https://firebase.google.com/docs/rules/basics

**Comment by colbySullivan** [comment]
Just realized I forgot the picture for the rules above.
![image](https://github.com/MuellMark/Course-Scheduler/assets/88957296/79cf7da6-99e2-4875-9ccc-ecf04bac8865)

**Comment by colbySullivan** [comment]
For the last 30 minutes or so I have been researching how to accurately incorporate Userid checking into our database if we do decide to secure it like that. I have also looked into to automating backups as we do do not have a system set up yet. Lastly I looked into restructuring our database so that classes would be organized by course type but I will wait until Monday to finalize that sections.
https://firebase.google.com/docs/database/backups
